android/app/src/main/java/org/solseed/thepeopleofgaia/ui/home/HomeViewModel.java

```diff
@@ -0,0 +1,19 @@
+package org.solseed.thepeopleofgaia.ui.home;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+public class HomeViewModel extends ViewModel {
+
+    private MutableLiveData<String> mText;
+
+    public HomeViewModel() {
+        mText = new MutableLiveData<>();
+        mText.setValue("This is home fragment");
```

Comment:
Can we get this improved in the next PR?

Reply:
Let's discuss what we want on our home page.